ArdentHQ/arkscan
# feat: schedule reindex wallets

## Summary

- With this change the wallets are going to be reindexed constantly
- If a previous job is running it will not reindex until no job is running


### Testing

1. run `php artisan schedule:run`  (this is the command that runs every minute on the server to check for schedules jobs)
2. should run the `scout:import` command (in the logs)
3. Since it takes time, the `schedule:run` command is not goin to end immediatly
4. wait > 1 minute and in other tabs run `php artisan schedule:run` again, the `scout:import` schedule job  should not run again
5. when the first job ends, if you run `php artisan schedule:run` it should run the `scout:import` job again

This job should be tested on a real server


## Checklist
